### PR TITLE
Fixed submodule script

### PR DESCRIPTION
**Pull request by DragonDev07**
FIxed the submodule.sh script stored in the "scripts" directory, so that it actually works.

### CODE REVIEW

**Comment by IsaccBarker** [comment]
Would you mind rebasing all your commits into one commit? For the sake of the commit history...

File: scripts/submodule.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 echo "Pulling submodules..."
-set -r
-git submodule update --recursive --remote
 
+# Initialize the submodules and then update
+git submodule init
+git submodule update --recursive
```
Comment (IsaccBarker) [inline scripts/submodule.sh:5]: Add the `--remote` flag, as seen in the line that you deleted.

File: .gitignore [hunk 2]
```diff
@@ -88,3 +88,7 @@ lint/tmp/
 
 # Annoying macOS Finder metadata
 **/.DS_Store
+
```
Comment (IsaccBarker) [inline .gitignore:91]: Remove this newline.

**Review by IsaccBarker — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (DragonDev07) [reply .gitignore:91]: Sorry, That was because I was trying to add in the .vscode folder but forgot lmao

**Review by IsaccBarker — APPROVED** [review]
Great work!

**Comment by IsaccBarker** [comment]
WAIT!

**Comment by IsaccBarker** [comment]
Would you mind rebasing all your commits into one commit? For the sake of the commit history...

**Review by IsaccBarker — CHANGES_REQUESTED** [review]
Would you mind rebasing all your commits into one commit? For the sake of the commit history...

**Comment by DragonDev07** [comment]
> 

No, it doesnt matter, this is the pre-season repo anyway. We are going to make separates for teams within the next 2 weeks probably.

**Review by IsaccBarker — APPROVED** [review]
Fine.

**Pull request merged**